manifests/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Don't commit this file, it's autogenerated by `kustomize`

Reply:
this was previously committed, I can remove it

Reply:
not sure if you need something from this file. if you do, we shouldn't commit values specific for your namespace like `registry.cloud.okteto.net/gabriel-okteto/todo-list`

Reply:
removed